static_src/stores/org_store.js

```diff
@@ -1,15 +1,16 @@
-
-/*
```

Comment:
is there a reason you got rid of this?

Reply:
@jcscottiii I will revert - I just removed it because it didn't really tell us anything we don't already know: `org_store.js` is a store for org data :)